rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n2.yaml

```diff
@@ -0,0 +1,25 @@
+---
+###############################################################################
+# SR-IOV Network Definition for the NS of the workload
+#  N2-bh + vlan 903 + core_sriov_netdevice
+###############################################################################
+# NS = open5gs-core
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: n2bh
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 903
+  ipam: |-
+    {
+      "type": "static",
+      "addresses": [
+        {
+          "address": "10.28.3.2/24"
```

Comment:
10.28 address here confuses me.  vlan 903 is also not defined anywhere in the lab.  am i missing something?

Reply:
We need more IP network addresses. We only have created VLAN and IP network for 4G flavour. Need additional ones to decouple the 4G from the 5G flavour.